packages/react-utilities/src/utils/properties.ts

```diff
@@ -217,17 +197,17 @@ export const buttonProperties = toObjectMap(htmlElementProperties, [
 ]);
 
 /**
- * An array of INPUT tag properties and events.
- *
- * @public
+ * A set of INPUT tag properties and events.
  */
-export const inputProperties = toObjectMap(buttonProperties, [
+export const inputProperties = new Set<keyof React.InputHTMLAttributes<{}> | CommonAttributes>([
+  ...buttonProperties,
   'accept', // input
   'alt', // area, img, input
   'autoCapitalize', // input, textarea
   'autoComplete', // form, input
   'checked', // input
-  'dirname', // input, textarea
+  'defaultChecked',
+  'defaultValue',
```

Comment:
Should we call where these appear as well?

```suggestion
  'defaultChecked', // input
  'defaultValue', // input, select
```

Reply:
I started doing this, but thinking about it more, I'm not sure we should have the comments at all--some of them are inaccurate or incomplete, and they don't serve much purpose. Almost seems like a leftover from a previous implementation, or someone's notes while working on organizing things. So I might just delete them all instead 😂 (but open to feedback on that).

Reply:
I'm in favor of removing the comments. It's easy enough to look them up on MDN or in the HTML spec if you need to know more. For these React-specific ones it's probably worth calling out that they aren't real native props.